src/Entity/ServiceProvider.php

```diff
@@ -106,6 +106,20 @@ class ServiceProvider extends AbstractRole
     public $requestedAttributes;
 
     /**
+     * @var null|string
+     *
+     * @ORM\Column(name="url_en", type="string")
```

Comment:
Please check whether Doctrine `Column`s are actually `nullable` by default. May not cause any errors during runtime, but it's correct.